docs/source/notebooks/pcca.ipynb

```diff
@@ -4,17 +4,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Coarse-graining with PCCA+"
+    "# Coarse-graining with PCCA+\n",
+    "\n",
+    "Given a Markov state model describing a reversible Markov chain, i.e., given transition matrix $P\\in\\mathbb{R}^{n\\times n}$ with stationary distribution $\\pi$ it obeys the detailed balance condition\n",
+    "\n",
+    "$$ \\pi_i P_{ij} = \\pi_j P_{ji}, $$\n",
+    "\n",
+    "its transition matrix can be coarse grained into metastable sets using the spectral clustering algorithm \"Robust Perron Cluster Analysis\" (PCCA+) <cite data-cite=\"nbpcca-roblitz2013fuzzy\">(Röblitz, 2013)</cite>. The result of the clustering is a coarse-grained transition matrix\n",
```

Comment:
i think this sentence needs a grammatical revisit

Reply:
reformulated